source/code/plugins/out_oms_changetracking_file.rb

```diff
@@ -0,0 +1,591 @@
+module Fluent
+
+  class OutChangeTrackingFile < BufferedOutput
+
+    Plugin.register_output('out_oms_oms.changetracking.file', self)
+	
+    # Endpoint URL ex. localhost.local/api/
+
+    def initialize
+      super
+	  
+      require 'base64'
+      require 'digest'
+      require 'json'
+      require 'net/http'
+      require 'net/https'
+      require 'openssl'
+      require 'rexml/document'
+      require 'securerandom'
+      require 'socket'
+      require 'uri'
+      require_relative 'omslog'
+      require_relative 'oms_configuration'
+      require_relative 'oms_common'
+    end
+
+    config_param :omsadmin_conf_path, :string, :default => '/etc/opt/microsoft/omsagent/conf/omsadmin.conf'
+    config_param :cert_path, :string, :default => '/etc/opt/microsoft/omsagent/certs/oms.crt'
+    config_param :key_path, :string, :default => '/etc/opt/microsoft/omsagent/certs/oms.key'
+    config_param :proxy_conf_path, :string, :default => '/etc/opt/microsoft/omsagent/proxy.conf'
```

Comment:
As I noted before, please check that these paths are all correct in the plugin files for default/non-multi-homing scenarios. You should also consider that these will have to be changed to be multi-homing compatible fairly soon.